README.md

```diff
@@ -177,11 +177,10 @@ campaign.set_lifetime_budget(
 
 **Note**: More examples of usage are located in the ``examples/`` folder.
 
-## Documentation For Models
+## Documentation
 
-* [pinterest ads](https://github.com/pinterest/pinterest-python-sdk/blob/main/docs/markdown/pinterest.ads.md)
+* [Overview](https://github.com/pinterest/pinterest-python-sdk/blob/main/docs/pinterest/README.md)
```

Comment:
I think it would be better if we kept a bulleted list of each module as it will be easier for a user to navigate. Is there a way to do that? We could redirect them to anchors to the doc page on the devsite too.

Reply:
Redirect them into the devsite is better. Let's keep it this way and I'll update it when devsite is available

Reply:
If we are not sending them tot he devsite then lets keep each component separate